src/test/java/org/jabref/gui/externalfiles/ImportHandlerTest.java

```diff
@@ -109,7 +108,13 @@ void cleanUpEntryTest() {
         BibEntry entry = new BibEntry().withField(StandardField.AUTHOR, "Clear Author");
         BibEntry cleanedEntry = importHandler.cleanUpEntry(bibDatabaseContext, entry);
 
-        assertEquals("Clear Author", cleanedEntry.getField(StandardField.AUTHOR).orElse(""));
+        Optional<String> authorOptional = cleanedEntry.getField(StandardField.AUTHOR);
+
+        // Make sure the value exists in Optional
+        assertTrue(authorOptional.isPresent());
+
+        // Get and verify the actual value in Optional
+        assertEquals("Clear Author", authorOptional.get());
```

Comment:
Just use following one liner:

```suggestion
        assertEquals(Optional.of("Clear Author"), authorOptional);
```